aspnetcore/fundamentals/websockets.md

```diff
@@ -123,6 +137,33 @@ The code shown earlier that accepts the WebSocket request passes the `WebSocket`
 
 When accepting the WebSocket connection before beginning the loop, the middleware pipeline ends. Upon closing the socket, the pipeline unwinds. That is, the request stops moving forward in the pipeline when the WebSocket is accepted. When the loop is finished and the socket is closed, the request proceeds back up the pipeline.
 
+::: moniker range=">= aspnetcore-2.2"
+
+### WebSocket Origin Restriction
```

Comment:
Lowercase origin restriction

sentence case headings; lowercase "origin" except when it refers specifically to "Origin" as the name of a header.